exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/exporter.py

```diff
@@ -194,9 +196,19 @@ def __init__(
         super().__init__()
 
         endpoint = endpoint or environ.get(
-            OTEL_EXPORTER_OTLP_ENDPOINT, "localhost:4317"
+            OTEL_EXPORTER_OTLP_ENDPOINT, "http://localhost:4317"
         )
 
+        parsed_url = urlparse(endpoint)
+
+        if endpoint is None:
```

Comment:
did you intend to write `is not` here? Because there is a default value and I don't see how it can be `None`? Could you also add a test to insecure_channel?

Reply:
thank you for catching this, i meant to check for `insecure`, there's already a few tests for the insecure_channel but i can add one specifically with the endpoint parameter.

Reply:
test added if you can take a look @lonewolf3739